hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -28,7 +29,15 @@
 @Category(QuickTest.class)
 public class LocalMapStatsTest extends HazelcastTestSupport {
 
+    /**
```

Comment:
Isn't it better to change the tests so they reflect the fact that not all information retrieved even though it is present.